components/Auth/TokenAuthorization.js

```diff
@@ -62,11 +62,15 @@ class TokenAuthorization extends Component {
   }
 
   goTo = (type, email, context) => {
+    const { router } = this.props
     if (this.props.goTo) {
       this.props.goTo(type, email, context)
       return
     }
-    Router.replaceRoute('notifications', { type, email, context })
+    router.replace({
+      pathname: '/benachrichtigung',
```

Comment:
```suggestion
      pathname: '/mitteilung',
```

There is no route `/benachrichtigun`. It should be `/mitteilung` due to renaming pages/notifications.js to pages/mitteilung.js

Reply:
😄 🎣 good catch and thanks!